packages/twenty-front/src/modules/object-record/hooks/useFindManyRecords.ts

```diff
@@ -126,111 +98,24 @@ export const useFindManyRecords = <T extends ObjectRecord = ObjectRecord>({
       },
```

Comment:
Line 97 above is the cause of the errors getting thrown by the two actions `front-sb-build` and `front-task (typecheck)`, however, notice that we do not make any changes to it to cause this error!!

![Screenshot 2024-06-11 at 18 02 09](https://github.com/twentyhq/twenty/assets/140154534/2a54db0b-2e4d-4766-b8ac-48c0771d9575)